### DISCUSSION TITLE
Replacing iris with iris starline

### DISCUSSION THREAD

**Discussion by AryanShetty** [IrisShaders/Iris#979] [Q&A] [answered]
Hi there! I've been using iris for quite a while now, and have finally decided to move to iris starline. I used the iris installer to install iris, and thats why I'm a little confused as to what exactly I need to do to make the move. It doesn't seem like I can simply replace the files because there is a single file called iris and sodium build 4.

The same file exists in two seperate locations. The mods folder, and another folder .in the .minecraft directory called iris-reserved. Do i need to delete that? or can i keep it?

**Comment by AlsoSylv** [comment, marked as answer]
Starline is an unsupported fork of iris.

**Reply by AryanShetty** [reply]
> Starline is an unsupported fork of iris.

What do you mean by that? Will nobody be able to help me here? or are you just giving me some information?

**Reply by BeansBeefBroccoli** [reply]
Ok uh, yea probably not here, try making a discussion in their repo.

**Reply by NoComment1105** [reply]
Starline recieves no support from the official Iris team

**Reply by enagimintas** [reply]
Why did you install official Iris and on the official iris you try to install starline? If you need other version of Iris or sodium, for example Starline, then delete all files connected to Iris and Sodium in .mods folder and install Starline
